### ISSUE TITLE
Add Maven archetype

### ISSUE THREAD

**Issue by ripla** [vaadin/platform#603]
* `versions.json`
* release notes

**Comment by ZheSun88** [comment]
the archetypes version should following the platform version, the last release is here https://repo.maven.apache.org/maven2/com/vaadin/vaadin-archetype-application/13.0.2/ (more instructions in the release check list.)

if you want to list the contents for the archetypes, currently, we only have `vaadin-archetype-application` one.

**Comment by pleku** [comment]
I have absolutely no idea why it should be in `versions.json` since it will follow the released platform version...

But I would want to include the archetype(s) in this part:

```
Getting Started with Vaadin 13
The best way to get started with Vaadin 13 is to go to https://vaadin.com/start and pick an app template for the technology stack you’re interested in. There are two types of app templates available.

The Project Bases are for starting your project from scratch with only the necessary dependencies and a couple of placeholder files available.

There are also full application examples available like Bakery (Pro) and Beverage Buddy. Those show you some opinionated examples on how to build different types of applications, with optionally integrating to a backend.

Getting Started Manually
For frontend projects you can get the dependencies with Bower or NPM.

Bower by running bower install vaadin#13.0.2 or bower install vaadin-core#13.0.2
NPM by running npm install @vaadin/vaadin@13.0.2 or npm install @vaadin/vaadin-core@13.0.2
There are also version-locked (with shrinkwrap) NPM dependencies available from @vaadin/vaadin-shrinkwrap#13.0.2 and @vaadin/vaadin-core-shrinkwrap#13.0.2.
For Java projects, an example of the necessary setup can be found from the Project Base.
```

Basically it there should be added something like:


#### Maven Archetypes

Maven is the de-facto build tool for Java web applications. Major IDEs also support Maven out of the box and most often you'll be using Maven via your favorite IDE. There is currently one Maven archetype available, the `vaadin-archetype-application` which corresponds to the project base for Flow. The version of the archetype should match the platform version. If you have Maven already installed, you can quickly create and run a Vaadin app with the following command:
```
mvn -B archetype:generate \
		-DarchetypeGroupId=com.vaadin \
		-DarchetypeArtifactId=vaadin-archetype-application \
		-DarchetypeVersion=##PLATFORM.VERSION##\
		-DgroupId=org.test \
		-DartifactId=vaadin-app \
		-Dversion=1.0-SNAPSHOT
		cd vaadin-app
		mvn package jetty:run
```

**Comment by ripla** [comment]
@pleku thanks for the update, didn't realise it would be following the archetype version.
